orc8r/cloud/go/obsidian/config.go

```diff
@@ -22,7 +22,8 @@ const (
 	DefaultServerCertKey         = "server_cert.key.pem"
 	DefaultClientCAs             = "ca_cert.pem"
 	DefaultStaticFolder          = "/var/opt/magma/static"
-	StaticURLPrefix              = "/apidocs"
+	StaticURLPrefix              = "/swagger/v1"
```

Comment:
This is actually just `/swagger`. Will be important for permissions for reading Swagger static assets